.github/workflows/template-tests.yml

```
@@ -0,0 +1,39 @@
+      - '.github/workflows/template-tests.yml'
+
+jobs:
+  test-starter-projects:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Python 3.12
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)